java/README.md

```diff
@@ -0,0 +1,109 @@
+# Semantic Kernel for Java
+
+Semantic Kernel (SK) is a lightweight SDK that lets you easily mix conventional programming languages with the latest in Large Language Model (LLM) AI "prompts" with templating, chaining, and planning capabilities out-of-the-box.
+
+To learn more about Microsoft Semantic Kernel, visit the [Microsoft Semantic Kernel documentation](https://learn.microsoft.com/en-us/semantic-kernel/whatissk).
+
+The Semantic Kernel for Java is an SDK that implements the key concepts of the Semantic Kernel in Java. It is designed to be used in Java applications and services in an idiomatic way, and to be easily integrated with other Java libraries and frameworks.
+
+## Get Started
+
+To run the LLM prompts and semantic functions in this kernel, make sure you have an [Open AI API Key](https://openai.com/api/) or [Azure Open AI service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api).
+
+### Requirements
+
+To build the semantic kernel, you will need to have:
+
+- **Required**: 
+  - [JDK 17](https://microsoft.com/openjdk/) or newer
+  - [Apache Maven](https://maven.apache.org/) 3.9.1 or newer
+  - Azure OpenAI SDK for Java _(see below for instructions)_
+
+## Building the semantic kernel
+
+The semantic kernel is built using [Apache Maven](https://maven.apache.org/). To build the semantic kernel, you will first need to have the Azure OpenAI SDK for Java installed in your local Maven repository. At this moment, this SDK is not available on Maven Central, so you will need to build it locally from the source.
+
+### Build the Azure OpenAI SDK locally
+
+2. Clone the Azure SDK OpenAI repository
+
+       git clone git@github.com:Azure/azure-sdk-for-java.git
+
+3. Switch to the OpenAI feature branch
+
+       cd azure-sdk-for-java
+       git checkout feature/open-ai
```

Comment:
@johnoliver  Are you taking the action to change this when that feature branch get's merged?

Reply:
These instructions are out of date and not required, removing them